.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+      # run: cd backend && bun run lint
+
+    - name: Run Backend Tests
+      run: cd backend && bun test --coverage
+      # Add frontend tests if/when available
+      # run: cd frontend && bun test --coverage
+
+    - name: Verify Docker Build
+      # Assumes Dockerfile is at the root. Adjust path if needed.
+      run: docker build -t podpace:ci -f Dockerfile .
+      # Create a dummy Dockerfile if it doesn't exist yet to avoid failure
+      # Or comment this step out until Dockerfile is created in Phase 7
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)